app/src/main/java/io/apicurio/registry/ccompat/store/RegistryStorageFacade.java

```diff
@@ -17,39 +17,66 @@
 package io.apicurio.registry.ccompat.store;
 
 import io.apicurio.registry.ccompat.dto.Schema;
+import io.apicurio.registry.ccompat.dto.SchemaContent;
+import io.apicurio.registry.ccompat.rest.error.SchemaNotFoundException;
 import io.apicurio.registry.storage.ArtifactAlreadyExistsException;
 import io.apicurio.registry.storage.ArtifactNotFoundException;
 import io.apicurio.registry.storage.RegistryStorageException;
+import io.apicurio.registry.storage.RuleConfigurationDto;
 import io.apicurio.registry.storage.VersionNotFoundException;
+import io.apicurio.registry.types.RuleType;
 
 import java.util.List;
-import java.util.Set;
-import java.util.SortedSet;
 import java.util.concurrent.CompletionStage;
+import java.util.function.Function;
 
 /**
+ * Note: This facade is only expected to be used for AVRO artifact type.
```

Comment:
We need to check how they now support different types, and extend this facade accordingly.

Reply:
Agreed.